octobot_trading/personal_data/portfolios/portfolio_value_holder.py

```diff
@@ -222,14 +226,21 @@ def _recompute_origin_portfolio_initial_value(self):
 
     def _try_get_value_of_currency(self, currency, quantity, raise_error):
         """
-        try_get_value_of_currency will try to obtain the current value of the currency quantity in reference currency.
+        try_get_value_of_currency will try to obtain the current value of
+            the currency quantity in reference currency.
         It will try to create the symbol that fit with the exchange logic.
         :return: the value found of this currency quantity, if not found returns 0.
         """
         try:
             return self.convert_currency_value_using_last_prices(quantity, currency,
                                                                  self.portfolio_manager.reference_market)
         except errors.MissingPriceDataError as missing_data_exception:
+            try:
+                value = self._try_converting_with_multiple_pairs(currency, quantity)
+                if value:
```

Comment:
I think we should check `if converted_value is None` to allow 0 as a value

Reply:
done